akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala

```diff
@@ -16,8 +16,15 @@ import akka.util.Helpers.{ Requiring, ConfigOps, toRootLowerCase }
 import scala.concurrent.duration.FiniteDuration
 import akka.japi.Util.immutableSeq
 
-final class ClusterSettings(val config: Config, val systemName: String) {
+object ClusterSettings {
+  type Team = String
+  val TeamRolePrefix = "team-"
+  val DefaultTeam: Team = "default"
+
+}
```

Comment:
I don't have a strong opinion. It's good if we add useful helper methods so that string concatenation is not needed in user code, and then these could perhaps be internal?